workspaces/desktop/server/server.js

```diff
@@ -40,7 +40,7 @@ const options = cl_args(argument_option_defs);
 if (options.help) {
   const usage = cl_usage([{ header: 'Arguments', optionList: argument_option_defs }]);
   console.log(usage);
-  return;
+  process.exit(0);
```

Comment:
this is not a function; can't return.

Reply:
Sure we can, put this in a JS file and run it with Node:

```js
if (true) {
  return;
}

console.log('Hello, world!');
```

Reply:
Huh, TIL.